new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java

```diff
@@ -363,4 +823,267 @@ private void verifySampleAppAccessThroughNginx() {
+                    condition.getElapsedTimeInMS(),
+                    condition.getRemainingTimeInMS()))
+            .until(assertDoesNotThrow(() -> operatorRestServiceRunning(operNamespace),
+                "operator external service is not running"));
+      }
+      return opHelmParams;
+    }
+    return null;
+  }
+
+  /**
+   * Installs a Helm chart and expected to fail.
+   * @param helmParams the parameters to Helm install command like namespace, release name,
+   *                   repo url or chart dir, chart name
+   * @param chartValues the values to override in a chart
+   * @return error Message on success, null otherwise
+   */
+  public static String installNegative(HelmParams helmParams, Map<String, Object> chartValues) {
+    String namespace = helmParams.getNamespace();
+
+    //chart reference to be used in Helm install
+    String chartRef = helmParams.getChartDir();
+
+    getLogger().fine("Installing a chart in namespace {0} using chart reference {1}", namespace, chartRef);
+
+    // build Helm install command
+    String installCmd = String.format("helm install %1s %2s --namespace %3s ",
+        helmParams.getReleaseName(), chartRef, helmParams.getNamespace());
+
+    // if we have chart values file
+    String chartValuesFile = helmParams.getChartValuesFile();
+    if (chartValuesFile != null) {
+      installCmd = installCmd + " --values " + chartValuesFile;
+    }
+
+    // if we have chart version
+    String chartVersion = helmParams.getChartVersion();
+    if (chartVersion != null) {
+      installCmd = installCmd + " --version " + chartVersion;
+    }
+
+    // add override chart values
+    installCmd = installCmd + helmValuesToString(chartValues);
+
+    if (helmParams.getChartVersion() != null) {
+      installCmd = installCmd + " --version " + helmParams.getChartVersion();
+    }
+
+    // run the command
+    return getExecError(installCmd);
+
+  }
+
+  /**
+   * Executes the given command and return error message if command failed.
+   * @param command the command to execute
+   * @return Error message string for failed command or null if no failure
+   */
+  private static String getExecError(String command) {
+    getLogger().info("Running command - \n" + command);
+    ExecResult result = null;
+    try {
+      result = ExecCommand.exec(command, true);
+      getLogger().info("The command returned exit value: "
+          + result.exitValue() + " command output: "
+          + result.stderr() + "\n" + result.stdout());
+      if (result.exitValue() != 0) {
+        getLogger().info("Command failed with errors " + result.stderr() + "\n" + result.stdout());
+        return result.stderr();
+      }
+    } catch (Exception e) {
+      getLogger().info("Got exception, command failed with errors " + e.getMessage());
+      return result.stderr();
+    }
+    return null;
+  }
+
+  /**
+   * Scale domain .
+   */
+  private boolean scaleDomain(String domainNS, String domainUid,
```

Comment:
Check if we have a common utility to scale a domain.

Reply:
I needed utility to be boolean, it is just a wrap around existed

Reply:
This is a very commonly used action utility, you may consider creating a new override method with different signature/return type in common utility area, which can be used by other classed if needed.

Reply:
done